templates/product/product--mustache.html.twig

```diff
@@ -0,0 +1,42 @@
+{% verbatim %}
+  <a class="item-product--strikethrough item-product"
+        data-vars-product-name="{{ title }}"
+        data-vars-product-shop="{{ shop_name }}"
+        data-vars-product-price="{{ price }}"
+        data-vars-product-currency="{{ original_currency_code }}"
+        data-vars-product-uuid="{{ uuid }}"
+        data-vars-product-id="{{ provider_identifier }}"
+        data-vars-product-category="{{ category }}"
+        data-vars-product-sold-out="{{#stock_availability}}0{{/stock_availability}}{{^stock_availability}}1{{/stock_availability}}"
+        data-vars-product-stock-availability="{{stock_availability}}"
+        data-vars-product-provider="{{ provider }}"
+        data-vars-product-view-mode="amp"
+        data-vars-product-position="undefined"
+        href="{{#amp_shopping_link}}{{amp_shopping_link}}&#x3f;article={% endverbatim %}{{current_url}}&amp;title={{parent_title | url_encode(true)}}{% verbatim %}{{/amp_shopping_link}}{{^amp_shopping_link}}{{ shop_url }}&amp;subid={{ provider_identifier }}{{/amp_shopping_link}}"
```

Comment:
This might be nicer, extracted to a variable. Also not sure about the `verbatim/endverbatims` starting/stopping throughout strings.